aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/util/ResourceModelHelper.java

```diff
@@ -27,6 +28,13 @@ public static boolean shouldUpdateAfterCreate(final ResourceModel model) {
                 );
     }
 
+    public static boolean isRestoreToPointInTime(final ResourceModel model) {
+        // Parameters to rely on are UseLatestRestorableTime and RestoreTime to tell if this is a RestoreToPointInTime
+        // But SourceDBInstanceAutomatedBackupsArn and DbiResourceId are also unique identifying parameters of RestoreToPointInTime
+        return  BooleanUtils.isTrue(model.getUseLatestRestorableTime()) || StringUtils.hasValue(model.getRestoreTime())  ||
+                StringUtils.hasValue(model.getSourceDBInstanceAutomatedBackupsArn()) || StringUtils.hasValue(model.getDbiResourceId());
```

Comment:
What would be the expectation of both `getUseLatestRestorableTime` and `getRestoreTime` are missing?

Reply:
There are 2 errors that RDS can throw:
```
An error occurred (InvalidParameterCombination) when calling the RestoreDBInstanceToPointInTime operation: If UseLatestRestoreTime is not true, the RestoreTime parameter must be specified.
```
And
```
An error occurred (InvalidParameterValue) when calling the RestoreDBInstanceToPointInTime operation: You must enter one of the following parameters: SourceDBInstanceIdentifier, SourceDbiResourceId, or SourceDBInstanceAutomatedBackupsArn.
```

Reply:
Both of which are in `DEFAULT_DB_INSTANCE_ERROR_RULE_SET`

Reply:
I think in that case we need to either add all the properties back to condition (SourceDBInstanceIdentifier is missing), or not drop all 3 parameters